geomstats/geometry/euclidean.py

```diff
@@ -40,14 +46,17 @@ def random_uniform(self, n_samples=1, bound=1.):
 
         Parameters
         ----------
-        n_samples : int, optional
-            default: 1
-        bound : float, optional
-            default: 1.0
+        n_samples : int
+            Number of samples.
+            Optional, default: 1.
```

Comment:
no dot if its an integer -> 1

Reply:
Yes, but I think we want a full stop at the end of the description line?